processing/data_collection/gazette/spiders/sp_presidente_prudente.py

```diff
@@ -0,0 +1,53 @@
+import re
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpPresidentePrudenhteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3541406"
+    GAZETTE_URL = "https://www.gdoe.com.br/presidenteprudente/1"
+    allowed_domains = ["https://www.gdoe.com.br/", "gdoe.com.br"]
```

Comment:
Take a look in the warning that I got running the spider:
```
2020-09-29 00:56:40 [py.warnings] WARNING: /usr/local/lib/python3.7/site-packages/scrapy/spidermiddlewares/offsite.py:61: URLWarning: allowed_domains accepts only domains, not URLs. Ignoring URL entry https://www.gdoe.com.br/ in allowed_domains.
  warnings.warn(message, URLWarning)
```

I have not check the scrapy documentation yet. But I **think** you can just remote the fist item in the list. Can you double check that? Thanks!

Reply:
Yes, you are right. I guess this URL was there because of another method I tried before. Due to [docs](https://docs.scrapy.org/en/latest/topics/spiders.html#scrapy.spiders.Spider.allowed_domains) I will just remove the string in my next commit.